### PR TITLE
Header BO responsive (legacy part)

### PR DESCRIPTION
**Pull request by nihco2**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Header BO responsive (legacy part)
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/STAB-70
| How to test?  | Check with Lea's mockups

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/7439)

### CODE REVIEW

**Review by maximebiloe — APPROVED** [review]

**Comment by Shudrum** [comment]
@nihco2, have you fixed the problems?

Thank you!

**Comment by aleeks** [comment]
don't know why & how but on the product page 
product

**Comment by basma-yangui-prestashop** [comment]
It's OK now ;)

**Comment by aleeks** [comment]
Thank you @nihco2 @basma-yangui-prestashop

**Comment by aleeks** [comment]
Thank you @nihco2

**Comment by nihco2** [comment]
Thank you @basma-yangui-prestashop @Shudrum @aleeks @maximebiloe

File: admin-dev/themes/default/template/header.tpl [hunk 1]
```diff
@@ -216,26 +216,26 @@
...
-					<li id="employee_infos" class="dropdown">
+					<li id="employee_infos" class="dropdown hidden-xs">
 						<a href="{$link->getAdminLink('AdminEmployees')|escape:'html':'UTF-8'}&amp;id_employee={$employee->id|intval}&amp;updateemployee" class="employee_name dropdown-toggle" data-toggle="dropdown">
 							<span class="employee_avatar_small">
                 <img class="imgm img-thumbnail" alt="" src="{$employee->getImage()}" width="32" height="32" />
 							</span>
 						</a>
 						<ul id="employee_links" class="dropdown-menu">
-							<li>
+							<li data-mobile="true" data-from="employee_links" data-target="menu"
```
Comment (Shudrum) [inline admin-dev/themes/default/template/header.tpl]: Missing `>`!

Comment (Shudrum) [inline admin-dev/themes/default/template/header.tpl]: Ping @maximebiloe :trollface:

Comment (maximebiloe) [inline admin-dev/themes/default/template/header.tpl]: 🤕

**Pull request merged**